docs/reference-guide/13-file-format.md

```diff
@@ -18,6 +18,35 @@ This implies that, inside one of the previous categories, the number of availabl
 ### Short term storage groups
 STS groups in input are now "dynamic" : group names are no longer fixed by code, user is free to define these groups.
 
+In the "thematic trimming" section, the new dynamic variable `STS by group` is now used to enable/disable all variables (level/injection/withdrawal) for all groups. The following variables are obsolete and must not be provided
+
+```
+PSP_open_injection
+PSP_open_withdrawal
+PSP_open_level
+...
+
+Other1_injection
+Other1_withdral
+Other1_level
+...
+Other5_injection
+Other5_withdral
+Other5_level
+```
+(3*9=27 variables in total)
+
+The default value for group is "OTHER1".
+
+## Output
+### Hydro maximum generation/pumping power
+In existing directory `ts-numbers`, add sub-directory `hgp` containing TS numbers for hydro max pumping/generation, for each area.
+
+### ST Storage
+- Output columns for ST storage are capitalized (PSP_open_injection -> PSP_OPEN_INJECTION)
```

Comment:
First item could be replaced with : 
(for any STS group name **my_group**, 3 output columns are created : **MY_GROUP_INJECTION**, **MY_GROUP_WITHDRAWAL**, **MY_GROUP_LEVEL**)